packages/@atjson/renderer-commonmark/test/commonmark-test.ts

```diff
@@ -32,6 +32,13 @@ describe("commonmark", () => {
               "Image descriptions ![are escaped](example.jpg)",
           },
         },
+        {
+          id: "1",
+          type: "-atjson-parse-token",
+          start: 0,
+          end: 1,
+          attributes: {},
+        },
```

Comment:
What would happen here if we didn't have this parse token? We would see the replacement character after the image?

Reply:
Yeah, something like that. It's subtle bug, which was previously prevented by the renderer ignoring these cases.